### PR TITLE
feat: config file at `~/.aicommits`

### PR DESCRIPTION
**Pull request by privatenumber**
## Problem
Exporting private tokens to the shell environment is insecure because it can be accessed by any command from any directory.

Other tools like  npm doesn't accept the token as an environment variable: https://github.com/npm/npm/issues/8356

## Changes
- Follow npm's approach of having a config file in the home directory in [ini format](https://www.npmjs.com/package/ini) that can store the token, amongst other things in the future.
- Updated docs to include new instructions

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -1,4 +1,24 @@
-import { Configuration, OpenAIApi } from 'openai';
+import fs from 'fs/promises';
+import path from 'path';
+import os from 'os';
+import ini from 'ini'; import { Configuration, OpenAIApi } from 'openai';
```
Comment (yuler) [inline src/utils.ts:4]:
```suggestion
import ini from 'ini'; 
import { Configuration, OpenAIApi } from 'openai';
```

**Comment by Nutlope** [comment]
@privatenumber This PR looks great, thanks!

**Review by Nutlope — APPROVED** [review]
LGTM

**Comment by Nutlope** [comment]
@privatenumber Up to you on whether you want to merge this now then do the other PR to add the `config set` command or merge them both at the same time.

Regardless, like you said, I'll wait until they're both done before pushing a new npm release.

**Comment by yuler** [comment]
How about add `aicommits int` command for create `.aicommits`file

**Comment by privatenumber** [comment]
@yuler Good idea!

Thought about it some more and I think the scope of a config command is a little larger than I expected, and also not necessary enough to block a pretty useful feature.

I think this can be released as is.

**Pull request merged**